ui/src/components/OwnedItems.svelte

```diff
@@ -34,23 +34,24 @@
 						class="bg-[color:var(--color-tertiary)] text-[2rem] w-[4rem] h-[4rem] text-center aspect-square"
 						on:click={() => {
 							SendNUI('selectOwnedItem', item)
+							console.log(item.type)
 						}}
 					>
 						<i class="fa-solid fa-magnifying-glass" />
 					</button>
 					<div class="flex flex-col gap-2 w-full">
 						<h1 class="text-[2rem]">{item.label}</h1>
 					</div>
-                    {#if !item.type}
-                        <button
-                            class="bg-[color:var(--color-tertiary)] text-[2rem] py-4 h-full text-center aspect-square"
-                            on:click={() => {
-                                SendNUI('removeOwnedItem', item)
-                            }}
-                        >
-                            <i class="fa-solid fa-xmark" />
-                        </button>
-                    {/if}
+                    {#if item.type != 'storage' && item.type != 'clothing'}
```

Comment:
If we add more item types (which I'm sure we will), this won't be very extensible and we'll potentially need to keep adding all the item types here to exclude. 

@ImXirvin what was the original intention behind this check. It seems like its just to prevent them from removing the Wardrobe or Storage furniture items. Is that really necessary? I'm almost thinking we remove the conditional entirely and just show the remove furniture button on everything

Reply:
That is true, If Xirvin says the check isn't really necessary, I think it might be better removing that whole check!

Reply:
@complexza ya, lets just remove that check, make everything removeable.

if someone removes their stash, they'll just have to replace it again to access the items, but it shouldnt even have any effects of data loss since the stuff for creating/opening inventory is al separate, the storage items is just the means to open it